<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-05-28T14:36:42Z" name="InteBan" finished-at="2019-05-28T14:37:03Z" duration-ms="21149">
    <groups>
    </groups>
    <test started-at="2019-05-28T14:36:42Z" name="Banktest" finished-at="2019-05-28T14:37:03Z" duration-ms="21149">
      <class name="com.inetBank.testCases.TC_AddNew_Customer_003">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.inetBank.testCases.TC_AddNew_Customer_003@5ef60048]" started-at="2019-05-28T14:36:47Z" name="setUp" finished-at="2019-05-28T14:36:49Z" duration-ms="2010" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addNewCustomer()[pri:0, instance:com.inetBank.testCases.TC_AddNew_Customer_003@5ef60048]" started-at="2019-05-28T14:36:49Z" name="addNewCustomer" finished-at="2019-05-28T14:37:03Z" duration-ms="14026" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.inetBank.testCases.TC_AddNew_Customer_003@5ef60048]" started-at="2019-05-28T14:37:03Z" name="tearDown" finished-at="2019-05-28T14:37:03Z" duration-ms="87" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.inetBank.testCases.TC_AddNew_Customer_003 -->
      <class name="com.inetBank.testCases.TC_LoginTest_001">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.inetBank.testCases.TC_LoginTest_001@5af97850]" started-at="2019-05-28T14:36:42Z" name="setUp" finished-at="2019-05-28T14:36:45Z" duration-ms="3141" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginTest()[pri:0, instance:com.inetBank.testCases.TC_LoginTest_001@5af97850]" started-at="2019-05-28T14:36:45Z" name="loginTest" finished-at="2019-05-28T14:36:47Z" duration-ms="1630" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetBank.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.inetBank.testCases.TC_LoginTest_001@5af97850]" started-at="2019-05-28T14:36:47Z" name="tearDown" finished-at="2019-05-28T14:36:47Z" duration-ms="86" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.inetBank.testCases.TC_LoginTest_001 -->
    </test> <!-- Banktest -->
  </suite> <!-- InteBan -->
</testng-results>
